formats/keep/models/keep-json.ts

```diff
@@ -0,0 +1,50 @@
+export interface KeepListItem {
+	text: string;
+	isChecked: boolean;
+}
+
+export interface KeepAttachment {
+	filePath: string;
+	mimetype: string;
+}
+
+export interface KeepSharee {
+	isOwner: boolean;
+	type: string;
+	email: string;
+}
+
+export interface KeepLabel {
+	name: string;
+}
+
+export interface KeepJson {
+    createdTimestampUsec: number;
+	userEditedTimestampUsec: number;
+    //
+	isArchived?: boolean;
+	isPinned?: boolean;
+	isTrashed?: boolean;
+    //
+	title?: string;
+	textContent?: string;
+	listContent?: Array<KeepListItem>;
```

Comment:
Prefer to use `Type[]` instead of `Array<Type>`.

Reply:
Fixed and pushed